client/src/components/Disover.tsx

```diff
@@ -0,0 +1,116 @@
+import React, {FC, useState, useEffect} from "react";
+import axios from 'axios';
+import { Button, TextField } from '@material-ui/core';
+
+interface currentMovie {
+  id: number;
+  movie_id: string;
+  title: string;
+  release_date: string;
+  description: string;
+  trailer_url: string;
+  thumbnailUrl: string;
+  createdAt: string;
+  updatedAt: string;
+}
+
+
+const Discover:FC = (props: any) => {
+  const [currentMovie, setCurrentMovie] = useState<null | currentMovie >(null);
+  const [genresList, setGenresList] = useState([]);
+  const [directorsList, setDirectorsList] = useState([]);
+  const [actorsList, setActorsList] = useState([]);
+  const [movieList, setMovieList] = useState([]);
+
+
+
+  const handleNextClick = () => {
+    getRandomMovie();
+  }
+  const getGenresList = () => {
+    axios('http://localhost:3000/api/genres/')
```

Comment:
I'm pretty sure you can shorten the URL to just `/api/genres/`

Reply:
Agreed. You can run `axios.get('/api/genres/)`, which will help later so you won't need to go back and change each of these to 'https' when we get to that point.